contrib/systemd/system/podman-restart.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Podman Start All Containers With Restart Policy Set To Always
+Requires=podman.socket
+After=podman.socket
```

Comment:
I don't think the dependencies on `podman.socket` are needed.

Reply:
Oh, you're right!. My mistake. Thanks.